server/src/test/java/keywhiz/service/daos/SecretSeriesDAOTest.java

```diff
@@ -450,118 +460,141 @@ public void setCurrentVersion_failsWithIncorrectSecretContent() {
   }
 
   @Test public void getSecretSeriesByDeletedName() {
-    long now = OffsetDateTime.now().toEpochSecond();
-    long id = secretSeriesDAO.createSecretSeries("toBeFound_getSecretSeriesByDeletedName",
-        null, "creator", "", null, null, now);
-    long contentID = secretContentDAO.createSecretContent(id, "blah",
-        "checksum", "creator", null, 0, now);
-    secretSeriesDAO.setCurrentVersion(id, contentID, "creator", now);
-    secretSeriesDAO.softDeleteSecretSeriesById(id);
+    long secretSeriesID = createSoftDeletedSecretSeries("toBeFound_getSecretSeriesByDeletedName");
```

Comment:
nit: mind putting the secret name in a var? It removes line breaks and makes it easier on the 👀 

```
  @Test public void getSecretSeriesByDeletedName() {
    String secretName ="toBeFound_getSecretSeriesByDeletedName";
    long secretSeriesID = createSoftDeletedSecretSeries(secretName);

    List<SecretSeries> deletedSecretSeries =
        secretSeriesDAO.getSecretSeriesByDeletedName(secretName);
    assertThat(deletedSecretSeries.size()).isEqualTo(1);
    assertThat(deletedSecretSeries.get(0).name()).contains(secretName);
    assertThat(deletedSecretSeries.get(0).name()).isEqualTo(secretName);
    assertThat(deletedSecretSeries.get(0).id()).isEqualTo(secretSeriesID);
  }
 ```